### ISSUE TITLE
Metatarsal head osteochondrosis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11203]
**Preferred term label:**
Metatarsal head osteochondrosis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Foot x-ray showing metatarsal head fracture, subsidence, loose body formation, flattening, deformity, or resultant arthritis; Freiberg infarction; Freiberg's disease; osteonecrosis metatarsal head

UMLS has 3 terms for this for slightly different locations, but its listing for "broader concept" conflates Osteochondrosis with Osteochondritis so I haven't included that:
https://uts.nlm.nih.gov/uts/umls/concept/C3839566
https://uts.nlm.nih.gov/uts/umls/concept/C2109248
https://uts.nlm.nih.gov/uts/umls/concept/C2109249

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Freiberg's Disease

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Not sure I follow the suggested definition. Would the following be adequate?
Def: Avascular necrosis of a metatarsal head, most commonly involving the second, but also the third or fourth, metatarsal. 
Comment: Affected individuals may present with foot pain on weight-bearing, swelling, and tenderness. Radiological features include widening of the metatarsophalangeal joint space and flattening of the affected metatarsal head, at later stages metatarsal head sclerosis, cortical thickening, and intra-articular loose bodies. Osteochondrosis is a condition resulting from an epiphysis injury that alters enchondral ossification and produces irregularity at the joint surface. Freiberg disease is characterized as osteochondrosis of the metatarsal heads (primarily the second).
PMID:30685014
PMID:[30725993](https://pubmed.ncbi.nlm.nih.gov/30725993)

**Comment by MickeySegal** [comment]
I've asked the orthopedist how to respond.

**Comment by MickeySegal** [comment]
The orthopedist says that you definition is fine.

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001347
- label: Metatarsal head osteochondrosis
- parents: [Abnormal metatarsal morphology (HP:0001832)](https://hpo.jax.org/app/browse/term/HP:0001832)
- pmid: 
- def:  Avascular necrosis of a metatarsal head, most commonly involving the second, but also the third or fourth, metatarsal.
Comment: Affected individuals may present with foot pain on weight-bearing, swelling, and tenderness. Radiological features include widening of the metatarsophalangeal joint space and flattening of the affected metatarsal head, at later stages metatarsal head sclerosis, cortical thickening, and intra-articular loose bodies. Osteochondrosis is a condition resulting from an epiphysis injury that alters enchondral ossification and produces irregularity at the joint surface. Freiberg disease is characterized as osteochondrosis of the metatarsal heads (primarily the second).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11203
- synonyms: Freiberg disease(NONE)|Metatarsal head osteonecrosis(NONE)|Freiberg infarction(NONE)
